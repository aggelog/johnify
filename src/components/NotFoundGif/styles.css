.application-not-found__container {
    text-align: center;
    margin: 20px 0;
}