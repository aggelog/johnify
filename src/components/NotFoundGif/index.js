import React from 'react';

import './styles.css';

const NotFoundGif = () => (
  <div className="application-not-found__container">
    <h3>Sorry, couldn't found gif</h3>
    <h4>We are the best gif engine, but it needs to exist</h4>
  </div>
);

export default NotFoundGif;
