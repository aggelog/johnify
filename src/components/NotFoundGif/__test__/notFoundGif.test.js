import React from 'react';
import NotFoundGif from '../index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<NotFoundGif />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});