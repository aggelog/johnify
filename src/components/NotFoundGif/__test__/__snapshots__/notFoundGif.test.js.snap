// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="application-not-found__container"
>
  <h3>
    Sorry, couldn't found gif
  </h3>
  <h4>
    We are the best gif engine, but it needs to exist
  </h4>
</div>
`;
